raiden/lightclient/models/light_client_payment.py

```diff
@@ -26,10 +24,7 @@ def __init__(
         created_on: string,
         payment_status: LightClientPaymentStatus,
         identifier: string,
-        # this is only to get easy access to the client, we only query this from DB but is not needed
-        # to create a payment
-        sender_light_client_address: AddressHex = None,
-        receiver_light_client_address: AddressHex = None
+        light_client_address: Address
```

Comment:
We can't because it depends on the situation, if you receive a payment it's saved with the receiver lc address if you are sending is the initiator lc address, so i can't put a name to represent both cases, sender_or_receiver_light_address is not a good name actually so i keep it generic.

Reply:
ok, so we use the same object for sent and received payments? wouldn't the address in that case always be the sender of the payment?

Reply:
i don't think i get it yet :thinking: 

i find these two statements contradictory:
> if you receive a payment it's saved with the receiver lc address 

>if you are sending is the initiator lc address

if the address is set to the initiator when you send the payment, how can it then be set to the receiver when you receive it?

Reply:
Ok so i got confussed about this, and you are right, it's always the initiator so i will change that

Reply:
thanks!

Reply:
If I don't getting it wrong, we are now duplicating the messages: The hub store messages when they are sent, and when they are received, right?